examples/simple_feature_set/simple_feature_set.ipynb

```diff
@@ -89,7 +89,7 @@
     "| - | - | - | - | - | - | - | - | - | - | - | - | - | - |\n",
     "| int | timestamp | float | float | int | int | float | float | float | double | double | string | string | string |\n",
     "\n",
-    "For more information about H3 geohash click [here]()\n",
+    "For more information about H3 geohash click [here](https://h3geo.org/docs/comparisons/geohash)\n",
```

Comment:
```suggestion
    "For more information about H3 geohash click [here](https://h3geo.org/docs/)\n",
```

Reply:
Hey thank you for the reply, I thought it was a comparison between S2 and H3 that author was intending to point out here. I will change it. Thanks :)